solr/test-framework/src/java/org/apache/solr/SolrIgnoredThreadsFilter.java

```diff
@@ -52,7 +56,23 @@ public boolean reject(Thread t) {
     }
     
     // These is a java pool for the collection stream api
-    if (threadName.startsWith("ForkJoinPool.")) {
+    if (threadName.startsWith("ForkJoinPool.") || threadName.startsWith("pool-")) {
+      return true;
+    }
+
+    // load balancer is leaky
+    if (threadName.startsWith("aliveCheckExecutor")) {
+      return true;
+    }
+
+
+    // we don't handle zk shutdown well, but these threads are harmless and will shortly go away
+    if (threadName.startsWith("SessionTracker")) {
+      return true;
+    }
+
+    // tools
+    if (threadName.startsWith("Reference Handler") && threadName.startsWith("Signal Dispatcher") && threadName.startsWith("Monitor") && threadName.startsWith("YJPAgent-RequestListener")) {
```

Comment:
Some of these might be useful to commit to randomized testing directly? Like I did in https://github.com/randomizedtesting/randomizedtesting/pull/284